acceleration-everest-03-dotnet/DomainService/Services/CustomerService.cs

```diff
@@ -68,7 +68,7 @@ public async Task<IEnumerable<Customer>> GetAllCustomersAsync()
 
     public async Task<Customer> GetCustomerByIdAsync(long id)
     {
-        var repository = _repositoryFactory.Repository<Customer>();
+        var repository = _unitOfWork.Repository<Customer>();
         var query = repository.SingleResultQuery().AndFilter(customer => customer.Id == id)
```

Comment:
Por que UnitOfWork em um get?
```suggestion
        var repository = _repositoryFactory.Repository<Customer>();
        var query = repository.SingleResultQuery().AndFilter(customer => customer.Id == id)
```

Reply:
Achei que precisava para fazer o teste funcionar, mas depois da ajuda do Claudinei percebi que era um erro meu mesmo.
Já passei para Repository Factory.